src/test/java/edu/cornell/kfs/fp/document/CuDisbursementVoucherDocumentTest.java

```diff
@@ -120,6 +121,11 @@ public static void setUp() throws Exception {
         cuDisbursementVoucherDocument.setDocumentHelperService(documentHelperService);
     }
 
+    @AfterClass
+    public static void tearDown() {
+        cuDisbursementVoucherDocument.setDocumentHelperService(null);
```

Comment:
Just trying to understand this fix.  Why was the documentHelperService the only service that had to be reset to null?